### PR TITLE
Java server to client streaming

### PR DESCRIPTION
**Pull request by mikaelm12**
Fixes: #9771 
Adding a section on the `hubConnection.stream` api for the java client

[Internal review link](https://review.docs.microsoft.com/en-us/aspnet/core/signalr/streaming?view=aspnetcore-2.1&branch=pr-en-us-11227)

### CODE REVIEW

File: aspnetcore/signalr/streaming.md [hunk 1]
```diff
@@ -133,6 +133,25 @@ To end the stream from the client, call the `dispose` method on the `ISubscripti
...
+* Arguments defined in the hub method. 
+
+```java
+hubConnection.stream(String.class, "ExampleStreamingHubMethod", "Arg1")
+    .subscribe(
+        (item) -> {/* Define your onNext handler here. */ },
+        (error) -> {/* Define your onError handler here. */},
+        () -> {/* Define your onCompleted handler here. */});
+```
+The `stream` method on `HubConnection` returns an Observable of the stream item type. The Observable types `susbscribe` method is where you define your `onNext`,  `onError` and  `onCompleted`
```
Comment (tdykstra) [inline aspnetcore/signalr/streaming.md]:
```suggestion
The `stream` method on `HubConnection` returns an Observable of the stream item type. The Observable type's `subscribe` method is where you define your `onNext`,  `onError` and  `onCompleted` handlers.
```

File: aspnetcore/signalr/streaming.md [hunk 2]
```diff
@@ -133,6 +133,25 @@ To end the stream from the client, call the `dispose` method on the `ISubscripti
 
 ::: moniker-end
 
+::: moniker range=">= aspnetcore-3.0"
+## Java client
+The SignalR Java client uses the `stream` method to invoke streaming methods. It accepts three arguments:
+
+* The expected type of the stream items 
+* The name of the hub method. In the following example we use an example server hub method.
```
Comment (tdykstra) [inline aspnetcore/signalr/streaming.md]:
```suggestion
* The name of the hub method.
```

File: aspnetcore/signalr/streaming.md [hunk 3]
```diff
@@ -133,6 +133,25 @@ To end the stream from the client, call the `dispose` method on the `ISubscripti
 
 ::: moniker-end
 
+::: moniker range=">= aspnetcore-3.0"
+## Java client
+The SignalR Java client uses the `stream` method to invoke streaming methods. It accepts three arguments:
```
Comment (tdykstra) [inline aspnetcore/signalr/streaming.md]: Should this say "three or more arguments", i.e., could the hub method take more than 1?

**Comment by mikaelm12** [comment]
Thanks for the review @tdykstra!

**Pull request merged**